docs/content/guides/developer/getting-started/graphql-rpc.mdx

```diff
@@ -367,6 +352,10 @@ where `$tx` and `$sigs` are your generated data, which may look like this:
 }
 ```
 
+## Other examples
+
+Other examples can also be found in the [repository](https://github.com/MystenLabs/sui/tree/main/crates/sui-graphql-rpc/examples), grouped into sub-directories. For example, there are directories for [transaction block effects](https://github.com/MystenLabs/sui/tree/main/crates/sui-graphql-rpc/examples/transaction_block_effects), [protocol configs](https://github.com/MystenLabs/sui/tree/main/crates/sui-graphql-rpc/examples/protocol_configs), [stake connection](https://github.com/MystenLabs/sui/tree/main/crates/sui-graphql-rpc/examples/stake_connection), and more.
```

Comment:
Since we're linking to main should we make a note that the examples may not be compatible with all versions of sui graphql a user may be trying them on? Do we have somewhere in the docs how a user can check compatibility

Reply:
Oh yes, that's a good point. Oh, this will be fun.

Reply:
Yeah I can add a piece about that!